source/slang/compiler.h

```diff
@@ -450,7 +452,6 @@ namespace Slang
 
         Dictionary<int, RefPtr<Type>> builtinTypes;
         Dictionary<String, Decl*> magicDecls;
-        List<RefPtr<Type>> canonicalTypes;
```

Comment:
What happens to `canonicalTypes`? I'm interested to see what the alternative is...